test/evergreen/import_compatibility_test.sh

```diff
@@ -21,11 +21,20 @@ build_branch()
 
     git checkout --quiet "$1"
 
-    config=""
-    config+="--enable-snappy "
-    config+="--disable-standalone-build "
-    (sh build_posix/reconf &&
-         ./configure $config && make -j $(grep -c ^processor /proc/cpuinfo)) > /dev/null
+    if [ "${build_sys[$1]}" == "cmake" ]; then
+            . ./test/evergreen/find_cmake.sh
+            config=""
+            config+="-DENABLE_SNAPPY=1 "
+            config+="-DWT_STANDALONE_BUILD=0 "
+            (mkdir build && cd build &&
```

Comment:
Question about consistency, sometimes we create the `build` folder (like here) and sometimes `cmake_build` for CMake builds (in evergreen.yml). Should we try to create a folder with the same name when it comes to CMake builds?

Reply:
So yep,  for builds created in the `evergreen.yml`, we generally use` cmake_build`. 
This specific script is a bit special however since its not run by the evergreen compilation task. Its checks out the wiredtiger repository multiple times at different revision points, running autoconf and/or cmake builds. To keep the logic and paths consistent, I wanted to establish a consistent build directory between a repository using an autoconf build vs a cmake build. Since this script is running compatibility tests against between checkouts e.g. a 5.0 checkout attempting to use 4.4 binaries, the logic is much simpler if they use identical paths.